templates/docs/upgrade-guide-v3.md

```diff
@@ -164,3 +164,7 @@ Horizontal spacing variables:
 | `$sph-inner--x-large` | Express as a sum of other vars | `2.5rem`      |
 | `$sph-outer`          | `$sph--small`                  | `0.5rem`      |
 | `$sph-outer--large`   | Express as sum of above vars   | `1.5rem`      |
+
+The variable `$max-width--default` has been renamed to `$text-max-width`. All other heading max-width variables have been replaced by $text-max-width.
+
+The keys in map `$nudges` no longer include the `$nudge--` prefix.
```

Comment:
The keys don't have `$` prefix:

```suggestion
The keys in map `$nudges` no longer include the `nudge--` prefix.
```

I know this would look quite repetitive, but could be useful to be more clear and explicit. Would be nice to have a table of all the renames:

| before | after |
| --- | ---|
| `map-get($nudges, nudge--p)` | `map-get($nudges, p)` |
| `map-get($nudges, nudge--h1)` | `map-get($nudges, h1)` |
| etc... | | 

with list of all renamed keys, to make it clear what gets replaced with what, and to make searching easier.

Reply:
I've pushed the fixes.